.github/workflows/rundoc.yml

```diff
@@ -0,0 +1,41 @@
+name: run-rundoc
+run-name: running rundoc on tutorials
+on:
```

Comment:
It would also be nice to trigger an execution when the `docs/src` folder is touched in a PR, then this will act as a test.

That being said, we probably don't also want it to send a PR, so we would likely want to bail out of that part. But it would be good to see the output.

Reply:
@lilacstella if you've got capacity. Prioritize the presentation first.